Samples/DeploymentManager/cs-winui/DeploymentManagerSample.csproj

```diff
@@ -10,13 +10,13 @@
     <UseWinUI>true</UseWinUI>
     <EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
     <PublishProfile>Properties\PublishProfiles\win10-$(Platform).pubxml</PublishProfile>
-	<WindowsSdkPackageVersion>10.0.19041.24</WindowsSdkPackageVersion>
+	<WindowsSdkPackageVersion>10.0.19041.27</WindowsSdkPackageVersion>
```

Comment:
We can probably drop this property, 27 should be the version in the .NET 6 SDK used by the pipeline.

Reply:
I'm going to do this separately.